src/main/java/org/humancellatlas/ingest/project/Project.java

```diff
@@ -43,6 +43,16 @@
     @Setter
     private Instant accessionDate;
 
+    @Setter
+    private Object technology;
+
+    @Setter
+    private Object dataAccess;
+
+    @Setter
+    private Object identifyingOrganisms;
```

Comment:
Do we know anything about these format of these fields apart from hat they are objects?

Reply:
Yeah, there's a schema defined in the UI but I'm not sure if i should apply it here. The info is not being used in core. And having Object makes it flexible to change like the content attribute. Thoughts? 


```
...
"dataAccess": {
      "$schema": "http://json-schema.org/draft-07/schema#",
      "description": "Data Access",
      "title": "Data Access",
      "name": "dataAccess",
      "type": "object",
      "properties": {
        "type": {
          "description": "Data Access",
          "guidelines": "What kind of consent do you have to share your data and metadata?",
          "type": "array",
          "items": {
            "type": "string"
          },
          "enum": [
            "All fully open",
            "All managed access",
            "A mixture of open and managed",
            "It's complicated"
          ],
          "user_friendly":  "Data Access"
        },
        "notes": {
          "description": "Data Access Notes",
          "type": "string",
          "user_friendly": "Data Access Notes",
          "guidelines": "Any data access notes?"
        }
      }
    },
    "identifyingOrganisms": {
      "description": "Identifying Organism",
      "guidelines": "What organ or organ system are you working on?",
      "type": "array",
      "items":{
        "type": "string"
      },
      "enum": [
        "Human",
        "Mouse",
        "Other"
      ],
      "user_friendly":  "Identifying Organism"
    },
    "technology": {
      "$schema": "http://json-schema.org/draft-07/schema#",
      "description": "Data Access",
      "title": "Data Access",
      "name": "dataAccess",
      "type": "object",
      "properties": {
        "ontologies": {
          "description": "The general methods used for sequencing library construction in your project.",
          "guidelines": "Enter the actual ontology id if you know it or keywords to search. If term is not found, please enter it in the other technologies field below.",
          "type": "array",
          "items": {
            "$schema": "http://json-schema.org/draft-07/schema#",
            "$id": "https://schema.dev.data.humancellatlas.org/module/ontology/1.2.5/library_construction_ontology",
            "description": "A term that may be associated with a process-related ontology term.",
            "additionalProperties": false,
            "required": [
              "text"
            ],
            "title": "Library construction ontology",
            "name": "library_construction_ontology",
            "type": "object",
            "properties": {
              "text": {
                "description": "The name of a library construction approach being used.",
                "type": "string",
                "user_friendly": "Library construction",
                "example": "10X v2 sequencing; Smart-seq2"
              },
              "ontology": {
                "description": "An ontology term identifier in the form prefix:accession.",
                "type": "string",
                "graph_restriction": {
                  "ontologies": [
                    "obo:efo"
                  ],
                  "classes": [
                    "OBI:0000711"
                  ],
                  "relations": [
                    "rdfs:subClassOf"
                  ],
                  "direct": false,
                  "include_self": false
                },
                "user_friendly": "Library construction ontology ID",
                "example": "EFO:0009310; EFO:0008931"
              },
              "ontology_label": {
                "description": "The preferred label for the ontology term referred to in the ontology field. This may differ from the user-supplied value in the text field.",
                "type": "string",
                "user_friendly": "Library construction ontology label",
                "example": "10X v2 sequencing; Smart-seq2"
              }
            }
          },
          "user_friendly": "Technology"
        },
        "others": {
          "description": "Other technologies not in the ontologies for technology.",
          "type": "array",
          "items": {
            "type": "string"
          },
          "user_friendly": "Other Technologies",
          "guidelines": "Other technologies not in the options above."
        }
      }
    },
...
```

Reply:
How do we intend to use these data?

Reply:
Long term we should definitely refer to this schema from a remote location, or have some way to say the type is "Object that follows a schema"

Definitely good for now. Will have a think.

Reply:
I'm not exactly sure how but the wranglers would like to capture and store this info. The technology & identifying organisms info i think will be used in spreadsheet template generation in the future.